src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/SaveTempDataPropertyFilter.cs

```diff
@@ -1,19 +1,31 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Generic;
 using System.Reflection;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Internal;
 
 namespace Microsoft.AspNetCore.Mvc.ViewFeatures.Internal
 {
-    public class SaveTempDataPropertyFilter : ISaveTempDataCallback
+    public class SaveTempDataPropertyFilter : ISaveTempDataCallback, IActionFilter
     {
-        public string Prefix { get; set; }
+        private readonly ITempDataDictionaryFactory _factory;
+        private const string Prefix = "TempDataProperty-";
```

Comment:
Typical field order we use in most places in our code: `public const`, `non-public const`, `public static readonly`, 'private static readonly`, `readonly`,  and finally non-readonly fields.